src/Socket.cpp

```diff
@@ -30,7 +30,7 @@
 #ifdef HAVE_LIBSSL
 // Copypasted from
 // https://wiki.mozilla.org/Security/Server_Side_TLS#Intermediate_compatibility_.28default.29
-// at 22 Dec 2014
+// at 2 Jan 2016
```

Comment:
Does it matter here?... It's not for machine, but for human anyway. And by spelling month with letters there's no ambuiguity of order of day and month